modules/con_notifications-configuration.adoc

```diff
@@ -92,6 +92,31 @@ spec:
 <1> Default name of the `NotificationsConfiguration` CR in a cluster.
 <2> An example custom service configuration for the `NotificationsConfiguration` CR.
 
+The following two examples are used for referencing the `oauth-access-token` in the `argocd-notifications-secret`. These steps are optional. 
```

Comment:
Yes, I agree with Max. Should the `oauth-access-token` and `argocd-notifications-secret` be added to the table? The example list could then follow the table order.

The table entries could mention that these two properties are optional. You can remove the "These steps are optional." just before the examples as it could be missed.

The module should be changed to a reference module.